core/Filesystem.php

```diff
@@ -30,6 +31,7 @@ public static function deleteAllCacheOnUpdate($pluginName = false)
         TrackerCache::deleteTrackerCache();
         PiwikCache::flushAll();
         self::clearPhpCaches();
+        ServerFilesGenerator::createFilesForSecurity();
```

Comment:
On my local server it takes like 2-3 minutes to execute this just FYI. Might slow down clearing caches. Might take a bit longer because I have also many files in `node_modules` but even without might take a while to execute